links/src/commonMain/kotlin/com/splendo/kaluga/links/repository/LinksRepo.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.links.repository
+
+import com.splendo.kaluga.links.manager.LinksManager
+import kotlinx.serialization.KSerializer
+
+class LinksRepo(
```

Comment:
I don't quite follow naming here: "links" + "repository" implies, that this entity stores links. But in fact, it doesn't really store anything.

Reply:
It comes from when it was using StateRepo, but from previous feedback (also out of band) this will have to be removed.